cheatsheets/Secrets_Management_Cheat_Sheet.md

```diff
@@ -211,6 +211,14 @@ A secret management solution should provide the capability to store at least the
 
 Note: if you don't store metadata about the secret nor prepare to move, you will increase the probability of vendor lock-in.
 
+### 2.12 Network segmentation of Secrets Management System
+
+Placing the Secret Management System in your own infrastructure means that all of its elements must be located in isolated networks (VLAN). Other network devices must not be located in networks where the Secret Management System servers are located. Each element (application) must be located in a separate suitable network according to the [Network Segmentation Cheat Sheet](https://github.com/OWASP/CheatSheetSeries/blob/master/cheatsheets/Network_Segmentation_Cheat_Sheet.md).
```

Comment:
Incorrect linking
```suggestion
Placing the Secret Management System in your own infrastructure means that all of its elements must be located in isolated networks (VLAN). Other network devices must not be located in networks where the Secret Management System servers are located. Each element (application) must be located in a separate suitable network according to the [Network Segmentation Cheat Sheet](Network_Segmentation_Cheat_Sheet.md).
```

Reply:
@szh validation build "Markdown Link Checker" not find that mistake. Is it possible to write rule to find same in next time?

Reply:
We should. It's a valid link, just not the desired location. It will always link to Github, even on the published site. Using a relative URL allows it to work both on Github and on the published site.

Reply:
It's a super common mistake though and we really should automate it